sample_data_config/surveymonkey/surveymonkey-data-pipeline.config.yaml

```diff
@@ -2,3 +2,6 @@ projectName: 'elife-data-pipeline'
 datasetName: '{ENV}'
 surveyMonkeySurveyList:
   table: 'surveymonkey_survey_list'
+  column: 'id'
```

Comment:
what is the meaning of that column?

Reply:
ah, that is for getting the `survey id` from bq to get the details for this specific survey

Reply:
okay, maybe it would benefit of a more specific name. e.g. `primary_key_column`?
(it may be worth renaming to `survey_id` for consistency, at some point)